lib/routes.js

```diff
@@ -24,7 +24,8 @@ const defaults = {
 		national:    true,
 		regionalExp: true,
 		regional:    true
-	}
+	},
+	bahncard: 		0 //25_1: 1, 25_2: 2, 50_1: 3, 50_2: 4
```

Comment:
Yes, I got it from here:
https://github.com/pajowu/db-python/blob/master/bahn.py
There are about 11 in total, in db-python only the major ones are documented. I'm gonna look up the others soon (Maybe its similar in the official api?)